### PR TITLE
docs: tweaks to the upgrade guide

### PR DESCRIPTION
**Pull request by helendurranttw**
## 🖼 Context

As per the title. Tweaks to the upgrade guide for clarity and to reduce duplication.

## ✅ Checklist

- [x] Make PR title meaningful and follow [the commit lint format](https://github.com/transferwise/neptune-web/blob/master/CONTRIBUTING.md#versioning-and-commit-lint) (it will appear in the changelog)
- [ ] Changes are tested and all tests pass
- [ ] Changes meet [accessibility standards](https://github.com/transferwise/neptune-web/blob/main/ACCESSIBILITY.md) and there are no violations in the console
- [ ] Changes work in all supported browsers (don't forget IE11)
- [x] You've updated the documentation if necessary

### CODE REVIEW

File: packages/docs/pages/blog/2021-11-30-november-release.tsx [hunk 1]
```diff
@@ -784,92 +537,319 @@ export default function PageContent(): ReactElement {
...
+        <Heading as="h2">2. Run the upgrade script</Heading>
+
+        <p>
+          Run the upgrade script, choose <code>2021-11-v11-v12.js</code> and follow the
+          instructions.
+          <pre>yarn run neptune-css-upgrade-util</pre>
+          <pre>npm exec neptune-css-upgrade-util</pre>
+        </p>
+
+        <Heading as="h2">3. Replace custom colours (recommended)</Heading>
```
Comment (dozortsev) [inline packages/docs/pages/blog/2021-11-30-november-release.tsx:778]: why we need "(recommended)" here? why we don't have "(recommended)" for 2. item?

**Review by dozortsev — COMMENTED** [review]
I would suggest to rename ["Semantic colours"](https://transferwise.github.io/neptune-web/branch/docs-tweaks/blog/2021-11-30-november-release#semantic-colours) header to "Semantic naming" because we talk about naming in that paragraph

**Review by dozortsev — COMMENTED** [review]
and please [remove image](https://github.com/transferwise/neptune-web/pull/1235#discussion_r761078488) 

it looks a lot better and cleaner without it

Screen Shot 2021-12-03 at 11 34 50 AM

File: packages/docs/pages/blog/2021-11-30-november-release.tsx (hunk 1 above)
Comment (helendurranttw) [reply packages/docs/pages/blog/2021-11-30-november-release.tsx:778]: Because it is recommended but not required

**Comment by helendurranttw** [comment]
Hey @dozortsev - I'm going to leave the image. I think it brings a bit of levity to a long document with lots of text :)

File: packages/docs/pages/blog/2021-11-30-november-release.tsx (hunk 1 above)
Comment (helendurranttw) [reply packages/docs/pages/blog/2021-11-30-november-release.tsx:778]: (E.g - their code won't break if they don't do it, but we would like them to do it)

**Review by dozortsev — COMMENTED** [review]
alright

**Review by dozortsev — APPROVED** [review]

**Pull request merged**